CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+- PII is automatically deleted from claims which are either rejected and have
+  had no update in the last two months, or which have been paid more than two
+  months ago.
```

Comment:
Conventionally we do not put fullstops on these entries unless they are more than one sentence.